docroot/sites/admissions.uiowa.edu/modules/admissions_core/admissions_core.module

```diff
@@ -407,6 +407,33 @@ function admissions_core_preprocess_field(&$variables) {
 function admissions_core_preprocess_field__node__title(&$variables) {
   if ($variables['element']['#bundle'] === 'major') {
     $variables['pre_title'] = AdmissionsCoreInterface::TWO_PLUS_TWO_TITLE;
+    // Remove existing page title classes.
+    $variables['attributes']['class'] = [];
+    // Add new page title classes.
+    uiowa_core_set_attributes($variables, [
+      'headline--uppercase',
+      'headline--negative',
+    ]);
+  }
+  if ($variables['element']['#bundle'] === 'area_of_study') {
+    // Remove existing page title classes.
+    $variables['attributes']['class'] = [];
+    // Add new page title classes.
+    uiowa_core_set_attributes($variables, [
+      'headline--uppercase',
+      'headline--negative',
+      'headline--highlight',
+    ]);
+  }
```

Comment:
This seems unrelated to this update.

Reply:
This fixes work we missed in https://github.com/uiowa/uiowa/pull/6203/files.